lib/KalturaHelpers.php

```diff
@@ -313,6 +313,21 @@ public static function getAllowedPlayers() {
 		return $players;
 	}
 
+	private static function _filterOldPlayers($players) {
+		$allowedPlayers = array();
+		foreach($players as $player) {
+			if(!empty($player->html5Url)) {
+				$htmlPlayerUrl = $player->html5Url;
+				$htmlPlayerUrlParts = explode('/', $htmlPlayerUrl);
+				if(isset($htmlPlayerUrlParts[3]) && substr($htmlPlayerUrlParts[3], 1, 1) === '2') {
```

Comment:
@rkreich - I thought about a different method for detecting HTML5 players, but I am not sure whether we can rely on a tag...
Besides, `html5v2` is an HTML5 player created in KMS... And there's the `html5studio` tag, but I am not sure that we can rely on it too.

Regarding the configuration - I consulted with Zohar and since Flash players do not behave correctly in some situations, he decided to drop the support for them...